setup.py

```diff
@@ -39,7 +39,7 @@
     "transformers>=2.5.0,<5",
     "bottleneck>=1.2.1,<2",
     "category_encoders>=1.3.0,<2",
-    "jinja2>=2,<3",
+    "jinja2>=2,<4",
```

Comment:
@anargyri, what do you think of adding this?

Reply:
As long as it passes the tests it's ok with me. This has the latest `staging` merged, hasn't it?